src/seth/libexec/seth/seth-index

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+### seth-index -- Prints the slot number for the specified mapping type and input data
+### Usage: seth index <fromtype> <totype> <fromvalue> <slot> [<lang>]
+###
+### Prints the slot number for the specified mapping type and input data. For example,
+### the balances mapping in DAI is slot 2, so to find the slot where the balance 
+### is stored for <account>, use
+###     seth-index address uint256 <account> 2
+
+set -e
+
+if [[ $5 = 'vyper' || $5 = 'vy' || $5 = 'v' ]]; then
+  # note: not guaranteed to work for all Vyper versions since storage layout is not yet stable
```

Comment:
Is it maybe worth putting some warning on std err here?

Reply:
Added in ba95b045